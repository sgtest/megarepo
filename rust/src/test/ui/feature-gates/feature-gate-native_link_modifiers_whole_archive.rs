#![allow(incomplete_features)]
#![feature(native_link_modifiers)]

#[link(name = "foo", modifiers = "+whole-archive")]
//~^ ERROR: `#[link(modifiers="whole-archive")]` is unstable
extern "C" {}

fn main() {}
