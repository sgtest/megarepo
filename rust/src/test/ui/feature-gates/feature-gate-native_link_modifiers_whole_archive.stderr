error[E0658]: `#[link(modifiers="whole-archive")]` is unstable
  --> $DIR/feature-gate-native_link_modifiers_whole_archive.rs:4:34
   |
LL | #[link(name = "foo", modifiers = "+whole-archive")]
   |                                  ^^^^^^^^^^^^^^^^
   |
   = note: see issue #81490 <https://github.com/rust-lang/rust/issues/81490> for more information
   = help: add `#![feature(native_link_modifiers_whole_archive)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
